slicerio/segmentation.py

```diff
@@ -689,7 +745,7 @@ def terminology_entry_matches(terminology1, terminology2):
 def segment_id_from_name(segmentation, segment_name):
     segments = segmentation["segments"]
     for segment in segments:
-        if segment_name == segments["name"]:
+        if segment_name == segment["name"]:
```

Comment:
this was probably wrong...?

Reply:
Yes, good catch

Reply:
Could you please also change `segments["id"]` below?

Reply:
done